### DISCUSSION TITLE
I can't seem to find this in the software...

### DISCUSSION THREAD

**Discussion by UES-GOV** [leozide/leocad#887] [Q&A]
If you don't have it could you add Brand Technic Panel Plate 11x19x1? 
Ref. Element | 6252630
Ref. Design | 39369
![image](https://github.com/leozide/leocad/assets/150491194/08d5a3df-2804-4003-889e-1cce97c95c5f)

**Comment by nathaneltitane** [comment]
you need to download and reference the ldraw parts library and have it set in your preferences - leocad only comes with the base library @NewHampshirelover2000 

[see here ](https://www.ldraw.org/)
